pkg/controller/build/build_controller.go

```diff
@@ -51,6 +51,11 @@ const (
 	desiredConfigLabel = "machineconfiguration.openshift.io/desiredConfig"
 )
 
+// on-cluster-build-custom-dockerfile ConfigMap name.
+const (
+	customDockerfileConfigMapName = "on-cluster-build-custom-dockerfile"
```

Comment:
Question: am I correct in understanding that this format will be provided via `poolName: osImageURL` in a list format? And incorrect formatted and unused mappings will simply be unused?

Reply:
It will be provided in a format more like `poolName: <Dockerfile>`. You are correct about missing or unused mappings being unused. In terms of correctness, a future TODO is to add some type of validation to that.